sql/world_updates/2017-01-29_01_scripttext.sql

```diff
@@ -0,0 +1,31 @@
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8761', 'Tiny creatures under feet, you bring Garfrost something good to eat!', '36494', '14', '16912');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8762', 'That one maybe not so good to eat now, stupied Garfrost! BAD! BAD!', '36494', '14', '16914');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8763', 'Will save.. for snack for.. for later!', '36494', '14', '16913');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8764', 'Garfrost hope giant underpants clean. Save boss great shame. For later.', '36494', '14', '16915');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8765', 'Axe too weak. Garfrost make better weapon and crush you!', '36494', '14', '16916');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8766', 'Garfrost tired of puny mortals, soon your bones will FREEZE!', '36494', '14', '16917');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8767', 'Our work must not be interrupted! Ick! Take care of them!', '36477', '14', '16926');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8768', 'Ooh...We could probably use these parts!', '36477', '14', '16927');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8769', 'Arms and legs are in short supply...Thanks for your contribution!', '36477', '14', '16928');
+INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES ('8770', 'Quickly! Poison them all while they're still close!', '36477', '14', '16930');
```

Comment:
Replace ' with \' inside the text, otherwise we run into sql errors, e.g.
`INSERT INTO `npc_script_text` (`entry`, `text`, `creature_entry`, `type`, `sound`) VALUES (8770, 'Quickly! Poison them all while they\'re still close!', 36477, 14, 16930);`